widgetry/src/widgets/mod.rs

```diff
@@ -255,7 +255,7 @@ impl Widget {
                 });
 
         // really short tab bodies look out of place in the panels
-        tab_body.layout.style.min_size.height = Dimension::Points(400.0);
+        tab_body.layout.style.min_size.height = Dimension::Points(200.0);
```

Comment:
This was a little excessive for the min height.

Here's what it looks like in the info-panels with the new height...
<img width="1072" alt="Screen Shot 2021-05-12 at 1 43 19 PM" src="https://user-images.githubusercontent.com/217057/118041895-37bb0800-b328-11eb-9452-c5641be8d329.png">

Maybe it'd look a little better a little taller? IDK... ultimately I think this just applies to the "info" tab which are going away once the panel redesigns happen.

Reply:
200 looks fine, and yeah, this use in the info panels isn't sticking around forever